scripts/test-end-to-end.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
Run the binary from the env:
```suggestion
#!/usr/bin/env bash
```
For bash this is not that big of a deal I guess, but it is better to use the system's env path instead of assuming hardcoding it.